pkg/cmd/pipeline/start.go

```diff
@@ -113,22 +135,21 @@ func startPipeline(out io.Writer, opt startOptions, p cli.Params, pName string)
 		return err
 	}
 
+	if len(res) > 0 {
+		pr.Spec.Resources = res
```

Comment:
I don't think that's right as it will override all the resource (requiring the user to provide all or nothing for resources). We need to merge the resources here (from last and provided)